core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/flow/blueprint/RDSConfigProvider.java

```diff
@@ -11,16 +12,39 @@
 @Component
 public class RDSConfigProvider {
 
-    public List<BlueprintConfigurationEntry> getConfigs(RDSConfig rdsConfig) {
+    public List<BlueprintConfigurationEntry> getConfigs(Set<RDSConfig> rdsConfigs) {
         List<BlueprintConfigurationEntry> bpConfigs = new ArrayList<>();
-        bpConfigs.add(new BlueprintConfigurationEntry("hive-site", "javax.jdo.option.ConnectionURL", rdsConfig.getConnectionURL()));
-        bpConfigs.add(new BlueprintConfigurationEntry("hive-site", "javax.jdo.option.ConnectionDriverName", rdsConfig.getDatabaseType().getDbDriver()));
-        bpConfigs.add(new BlueprintConfigurationEntry("hive-site", "javax.jdo.option.ConnectionUserName", rdsConfig.getConnectionUserName()));
-        bpConfigs.add(new BlueprintConfigurationEntry("hive-site", "javax.jdo.option.ConnectionPassword", rdsConfig.getConnectionPassword()));
-        bpConfigs.add(new BlueprintConfigurationEntry("hive-env", "hive_database", rdsConfig.getDatabaseType().getAmbariDbOption()));
-        bpConfigs.add(new BlueprintConfigurationEntry("hive-env", "hive_database_type", rdsConfig.getDatabaseType().getDbName()));
+        for (RDSConfig rds : rdsConfigs) {
+            switch (rds.getType()) {
+                case HIVE:
+                    bpConfigs.add(new BlueprintConfigurationEntry("hive-site", "javax.jdo.option.ConnectionURL", rds.getConnectionURL()));
+                    bpConfigs.add(new BlueprintConfigurationEntry("hive-site", "javax.jdo.option.ConnectionDriverName", rds.getDatabaseType().getDbDriver()));
+                    bpConfigs.add(new BlueprintConfigurationEntry("hive-site", "javax.jdo.option.ConnectionUserName", rds.getConnectionUserName()));
+                    bpConfigs.add(new BlueprintConfigurationEntry("hive-site", "javax.jdo.option.ConnectionPassword", rds.getConnectionPassword()));
+                    bpConfigs.add(new BlueprintConfigurationEntry("hive-env", "hive_database", rds.getDatabaseType().getAmbariDbOption()));
+                    bpConfigs.add(new BlueprintConfigurationEntry("hive-env", "hive_database_type", rds.getDatabaseType().getDbName()));
+                    break;
+                case RANGER:
+                    break;
+                case DRUID:
+                    bpConfigs.add(new BlueprintConfigurationEntry("druid-common", "druid.metadata.storage.type",
```

Comment:
@b-slim the connectionurl in database is jdbc:postgresql://druid.test.eu-west-1.rds.amazonaws.com:5432/druid so I think it is correct